README.md

```diff
@@ -183,7 +183,9 @@ be removed in the future.
 
 -   `mssql`: Add support for the MSSQL database server.
 
--   `sqlite`: Add support for the self-contained [SQLite](https://sqlite.org/) database engine.
+-   `sqlite`: Add support for the self-contained [SQLite](https://sqlite.org/) database engine with SQLite bundled and statically-linked.
+
+-   `sqlite-unbundled`: The same as above (`sqlite`) but use the SQLite shared library on the system instead of bundling a copy. This requires a recent version of SQLite library installed when building and running, and allows for updating the shared SQLite library for security fixes, replacing it with another compatible forks, or using SQLite's loadable extensions.
```

Comment:
Since this is already a bulleted list, I'd prefer to break this down into individual sub-points:

```suggestion
-   `sqlite-unbundled`: The same as above (`sqlite`), but link SQLite from the system instead of the bundled version.
    * Allows updating SQLite independently of SQLx or using forked versions.
    * You must have SQLite installed on the system or provide a path to the library at build time.
       See [the `rusqlite` README](https://github.com/rusqlite/rusqlite?tab=readme-ov-file#notes-on-building-rusqlite-and-libsqlite3-sys) for details.
    * May result in link errors if the SQLite version is too old.
```

I'm not sure where the "loadable extensions" part comes from. We already support that at runtime: https://docs.rs/sqlx/latest/sqlx/sqlite/struct.SqliteConnectOptions.html#method.extension

Unless you're talking about building a binary with the extensions statically compiled in, but that requires the application itself to invoke their entrypoints directly, or use `sqlite3_auto_extension()`: https://www.sqlite.org/loadext.html#statically_linking_a_run_time_loadable_extension

Reply:
> I'm not sure where the "loadable extensions" part comes from.

Sorry, I thought the symbols would not be visible because of static linking (and lto), but it isn't the case here. (I examined the command arguments but still don't know why.)